LoopKitHostedTests/InsulinDeliveryStoreTests.swift

```diff
@@ -208,7 +208,7 @@ class InsulinDeliveryStoreTests: InsulinDeliveryStoreTestsBase {
         XCTAssertEqual(entries[0].endDate, self.entry1.endDate)
         XCTAssertEqual(entries[0].value, 0.015)
```

Comment:
Adding a new `deliveredUnits` field to `CachedInsulinDeliveryObject` is probably the right approach for representing an interrupted bolus. Also, for basal doses this is probably an improvement over the old method of stashing a `DoseEntry`'s deliveredUnits into `value` and changing the `unit` to `.units`. So to be complete with this change, for basal doses, I think the `value` field here should now show the original programmed rate of 1.8, and the `unit` field should be `unitsPerHour`.  It also makes the `programmedTempBasalRate` field on `CachedInsulinDeliveryObject` redundant. I think that can also be dropped without needing a new model version.

Reply:
note that the new field is `programmedUnits`. The original `value` in the `CachedInsulinDeliveryObject` was just renamed to `deliveredUnits`.